lib/datasource/npm/get.ts

```diff
@@ -218,7 +218,12 @@ export async function getDependency(
       if (res.time?.[version]) {
         release.releaseTimestamp = res.time[version];
         release.canBeUnpublished =
-          moment().diff(moment(release.releaseTimestamp), 'days') === 0;
+          DateTime.local()
```

Comment:
Is `local` what we want here? I think it makes sense for schedules, but I would expect npm to be using utc?

Reply:
I'm not sure if it will make any difference, as I think luxon will repect timezone while diffing

Reply:
Okay, lets merge and see :crossed_fingers: